doc/reference/javascript_reference.rst

```diff
@@ -2149,6 +2149,19 @@ kanban_state_selection (StateSelectionWidget)
 
   - Supported field types: *selection*
 
+list.state_selection (ListStateSelectionWidget)
+  This is exactly the same widget as state_selection but display the label next to the icon
```

Comment:
In list views, the state_selection widget displays by default the label next to the icon.

Reply:
@aab-odoo , Done. thanks